core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/OracleExportAction.java

```diff
@@ -241,12 +236,23 @@ public void validate() {
       if (!containsMacro("oracleServerPort") && oracleServerSSHPort  < 0) {
         throw new IllegalArgumentException("Port cannot be negative");
       }
-      if (!EnumUtils.isValidEnum(SeparatorFormat.class, format)) {
+
+/*      EnumSet<SeparatorFormat> map =
+        new EnumSet<>(SeparatorFormat.class);*/
+      try{
+        Enum.valueOf(SeparatorFormat.class, format);
```

Comment:
why do you need this? Can you just do ``SeparatorFormat.valueOf(format.toUpperCase())``